01_install_requirements.sh

```diff
@@ -15,8 +15,10 @@ if [ -z "${METAL3_DEV_ENV}" ]; then
 fi
 
 pushd ${METAL3_DEV_ENV}
-# FIXME: Pin to aafbb89df8df88bc3ecbfe59592149c929432515
-git reset aafbb89df8df88bc3ecbfe59592149c929432515 --hard
+# Pin to a specific metal3-dev-env commit to ensure we catch breaking
+# changes before they're used by everyone and CI.
+# TODO -- come up with a plan for continuously updating this
+git reset b19da74e06062d0c21ade7dea1c6e8d09f7f4e48 --hard
```

Comment:
We should do this in the conditional L11..L14 - we only want to reset in the case where METAL3_DEV_ENV isn't set by a developer for testing a local checkout IMO.

Reply:
That can be a followup if you prefer but I noticed it trying to locally test with a newer metal3-dev-env so perhaps we can just move it in this PR?